specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -111,6 +111,9 @@ export function IntegratedRecordSelector({
 
   const [isDialogOpen, handleOpenDialog, handleCloseDialog] = useBooleanState();
 
+  const isTaxonTreeDefItemTable = 
+    collection.table.specifyTable.name.includes('TaxonTreeDefItem');
```

Comment:
would the following be equivalent?
if so, it would be a bit simpler:

```suggestion
    collection.table.specifyTable.name === 'TaxonTreeDefItem';
```